dummy_data_ingestion/obs_sur_dummy_data_ingest.py

```diff
@@ -0,0 +1,189 @@
+  "solutionName": "",
+  "scoringSystem": "",
+  "solutionDescription": "",
+  "questionSequenceByEcm": "",
+  "entityType": "",
+  "observationName": "",
+  "questionId": "",
+  "questionAnswer": "",
+  "questionResponseType": "",
+  "questionResponseLabel": "",
+  "questionExternalId": "",
+  "questionName": "",
+  "questionECM": "",
+  "criteriaId": "",
+  "completedDate": "",
+  "createdAt": "",
+  "updatedAt": "",
+  "remarks": "",
+  "totalEvidences": "",
+  "instanceParentQuestion": "",
+  "instanceParentId": "",
+  "instanceParentResponsetype": "",
+  "instanceId": "",
+  "instanceParentExternalId": "",
+  "instanceParentEcmSequence": "",
+  "channel": "",
+  "parent_channel": "",
+  "submissionNumber": "",
+  "submissionTitle": "",
+  "criteriaLevelReport": "",
+  "isRubricDriven": "",
+  "userProfile": ""
+}
+slObservationMeta = {
+    "completedDate": "",
+    "createdBy": "",
+    "entity": "",
+    "entityExternalId": "",
+    "entityType": "",
+    "isAPrivateProgram": "",
+    "observationName": "",
+    "observationSubmissionId": "",
+    "observationId": "",
+    "organisationName": "",
+    "solutionType": "",
+    "solutionExternalId": "",
+    "solutionId": "",
+    "solutionName": "",
+    "userType": "",
+    "userProfile": "",
+    "createdAt": ""
+}
+
+slObservationStatusStarted = {
+    "startedAt": "",
+    "observationSubmissionId": ""
+}
+
+slObservationStatusInprogress = {
+    "inprogressAt": "",
+    "observationSubmissionId": ""
+}
+
+slObservationStatusCompleted = {
+    "completedAt": "",
+    "observationSubmissionId": ""
+}
+
+slSurvey = {
+    "completedDate": "",
+    "createdAt": "",
+    "createdBy": "",
+    "criteriaExternalId": "",
+    "criteriaId": "",
+    "criteriaName": "",
+    "evidence_count": "",
+    "evidences": "",
+    "isAPrivateProgram": "",
+    "organisation_id": "",
+    "organisation_name": "",
+    "questionAnswer": "",
+    "questionECM": "",
+    "questionExternalId": "",
+    "questionId": "",
+    "questionName": "",
+    "questionResponseLabel": "",
+    "questionResponseLabel_number": "",
+    "questionResponseType": "",
+    "remarks": "",
+    "solutionExternalId": "",
+    "solutionId": "",
+    "solutionName": "",
+    "surveyId": "",
+    "surveyName": "",
+    "surveySubmissionId": "",
+    "total_evidences": "",
+    "updatedAt": "",
+    "user_type": ""
+}
+
+slSurveyMeta = {
+    "completedDate": "",
+    "createdBy": "",
+    "createdAt": "",
+    "isAPrivateProgram": "",
+    "organisationName": "",
+    "solutionExternalId": "",
+    "solutionId": "",
+    "solutionName": "",
+    "surveyName": "",
+    "surveySubmissionId": "",
+    "surveyId": "",
+    "userProfile": ""
+}
+
+slSurveyStatusStarted = {
+    "startedAt": "",
+    "surveySubmissionId": ""
+}
+
+slSurveyStatusInprogress = {
+    "inprogressAt": "",
+    "surveySubmissionId": ""
+}
+
+slSurveyStatusCompleted = {
+    "completedAt": "",
+    "surveySubmissionId": ""
+}
+
+def send_to_kafka(topic, data):
```

Comment:
added exception

Reply:
Capture exception in the method instead of handling for each method call 

def send_to_kafka(topic, data):
    try:
        producer.send(topic, value=data)
        producer.flush()
        print(f"Sent data to {topic}: {data}")
    except Exception as e:
        print(f"Failed to send data to {topic}: {e}")

Reply:
done @Vivek-M-08